docs/unique/ai-performance-optimization.md

```
@@ -0,0 +1,94 @@
+
+- Better Prompts for Local AI: ^ref-46e6b485-12-0
+
+- Additional Improvements:
+
+- Set up kitchen sink compose file
+  - Has vLLM for optimized batched LLM queries
+  - Has OpenVINO GenAI model server (NPU)
+
+[[vLLM]] [[OpenVINO]]
+
+#AI #Optimization #Parallelization #BatchProcessing #Memoizer #PromptEngineering
```

markdownlint-cli2 (0.17.2)

No space after hash on atx style heading

(MD018, no-missing-space-atx)